### PR TITLE
Closes AgileVentures/MetPlus_tracker#60

### PR DESCRIPTION
**Pull request by amba178**
This time, PR against development.

### CODE REVIEW

File: db/migrate/20151103015400_add_devise_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class AddDeviseToUsers < ActiveRecord::Migration
+  def self.up
+    change_table(:users) do |t|
```
Comment (patmbolger) [inline db/migrate/20151103015400_add_devise_to_users.rb]: Please add these two lines before change_table:

    change_column_null      :users, :email, false
    change_column_default :users, :email, ""

File: spec/factories/users.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 FactoryGirl.define do
   factory :user do
-  
-		first_name 'John'
+        password 'MyString1233'
+        first_name 'John'
 		last_name "Doe"
```
Comment (joaopapereira) [inline spec/factories/users.rb]: Indentation

**Comment by joaopapereira** [comment]
Please review the comments

File: db/migrate/20151103015400_add_devise_to_users.rb (hunk 1 above)
Comment (amba178) [inline db/migrate/20151103015400_add_devise_to_users.rb]: well, I understood. thanks.

**Comment by amba178** [comment]
fixed indentation
added:

change_column_null      :users, :email, false
 change_column_default   :users, :email, ""

**Comment by joaopapereira** [comment]
Looks Good

**Pull request merged**